awscli/customizations/wizard/ui/layout.py

```diff
@@ -64,3 +73,52 @@ def _create_section_tab(self, section_name, section_definition):
 
     def _create_section_body(self, section_name, section_definition):
         return WizardSectionBody(section_name, section_definition)
+
+
+class WizardDetailsPanel:
+    DIMENSIONS = {
+        'details_window_height_max': 40,
+        'details_window_height_pref': 30,
+    }
+
+    def __init__(self):
+        self.container = self._get_container()
+
+    def _get_container(self):
+        return ConditionalContainer(
+            HSplit([
+                TitleLine('Details panel'),
```

Comment:
Is the plan to put the dynamic title line (based on the details description) in a subsequent PR?

Reply:
Yes